velox/connectors/hive/tests/PartitionIdGeneratorTest.cpp

```diff
@@ -145,8 +145,32 @@ TEST_F(PartitionIdGeneratorTest, stableIdsMultipleKeys) {
   }
 }
 
+TEST_F(PartitionIdGeneratorTest, partitionKeysCaseSensitive) {
+  PartitionIdGenerator idGenerator(
+      ROW({"c0", "C1", "C2"}, {BIGINT(), VARCHAR(), INTEGER()}),
```

Comment:
When testing case sensitivity it is helpful to use values that have a mix of lower and upper case letters.

Reply:
@mbasmanova Yes. Updated.